code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -462,6 +462,9 @@
 		A.addMember(src)
 	SSair.add_to_rebuild_queue(src)
 
+/obj/machinery/atmospherics/update_name()
```

Comment:
```suggestion
/obj/machinery/atmospherics/proc/update_name()
```
Also add this to the multitool proc in the thermomachine code

Reply:
The multitool proc in the thermomachine code already already calls `update_appearance()`, which calls `update_name()`

Adding `proc/` here results in duplicate definition errors with `/atom/proc/update_name`:

```
code\modules\atmospherics\machinery\atmosmachinery.dm:462:error: update_name: duplicate definition
code\game\atoms.dm:687:error: update_name: previous definition
```

Of the entities with update_name defined, the following do not specify `proc/`:
 - `/obj/item/holochip/update_name`
 - `/obj/item/food/pancakes/update_name()`
 - `/obj/item/sharpener/update_name()`
 - `/obj/item/tcgcard/update_name(updates)`
 - `/obj/structure/sign/painting/update_name(updates)`
 - `/obj/structure/door_assembly/update_name()`
 - `/obj/structure/headpike/update_name()`
 - `/obj/structure/blob/normal/update_name()`
 - `/obj/structure/blob/shield/update_name(updates)`
 - `/obj/item/assembly/flash/update_name()`
 - `/obj/item/reagent_containers/food/drinks/sillycup/smallcarton/update_name(updates)`
 - `/obj/structure/chisel_message/update_name()`
 - `/obj/item/bot_assembly/floorbot/update_name()`
 - `/mob/living/simple_animal/slime/update_name()`
 - `/obj/item/stack/cable_coil/update_name()`
 - `/obj/item/stack/pipe_cleaner_coil/update_name()`
 - `/obj/item/reagent_containers/blood/update_name(updates)`
 - `/obj/item/organ/appendix/update_name()`
 - `/obj/item/mecha_ammo/update_name()`

The following do specify `proc/`:
 - `/atom/proc/update_name`
 - `/datum/team/brother_team/proc/update_name()`
 - `/datum/antagonist/ert/proc/update_name()`
 - `/datum/action/innate/integrate_extract/proc/update_name()`

These are all either datums or the base `/atom` definition

Reply:
ahhh i see, is not a newly made proc, my bad

Reply:
i saw the error on update_name and thought it was because it's missing the proc/ but in reality is just missing the call to parent

Reply:
Gotcha, fixed